src/groups/semidirect_product_group.jl

```diff
@@ -1022,9 +1033,44 @@ inv(
     SDPG::LieGroup{𝔽, <:SemidirectProductGroupOperation{O1, O2, A, AO}, <:ProductManifold}, ::Any,
 ) where {𝔽, O1, O2, A <: AbstractLeftGroupActionType, AO <: AbstractActionActsOnType}
 
+# function _inv!(
+#         SDPG::LieGroup{𝔽, <:SemidirectProductGroupOperation{O1, O2, A, AO}, <:ProductManifold}, k, g
+#     ) where {𝔽, O1, O2, A <: AbstractGroupActionType, AO <: AbstractActionActsOnType}
+#     if (A <: AbstractLeftGroupActionType) != (AO <: ActionActsOnLeft)
+#         _inv_semidirect_same!(SDPG, k, g)
+#     else
+#         _inv_semidirect_opposite!(SDPG, k, g)
+#     end
+# end
+
+function _inv!(
+        SDPG::LieGroup{𝔽, <:SemidirectProductGroupOperation{O1, O2, A, AO}, <:ProductManifold}, k, g
+    ) where {𝔽, O1, O2, A <: AbstractLeftGroupActionType, AO <: ActionActsOnRight}
+    return _inv_semidirect_forward_action!(SDPG, k, g)
```

Comment:
I never understood forward, so please let's not use this artificially introduced name here. It is not properly defined (in like I have zero references for this) and it just confuses readers. What did you actually change here?